server/channels/app/post.go

```diff
@@ -588,6 +591,14 @@ func (a *App) handlePostEvents(rctx request.CTX, post *model.Post, user *model.U
 		return err
 	}
 
+	// Send initial read receipt counts for burn-on-read posts
+	if post.Type == model.PostTypeBurnOnRead {
+		// No revealing user yet (post just created), send empty string
+		if err := a.publishPostRevealedEventToAuthor(rctx, post, "", ""); err != nil {
```

Comment:
1/5 - Can we include the connection ID here just to stay consistent with the model that we don't send WS events to the client generating the action? I see this would require changing the signature of `createPost` method so not worth the effort right now. But if connection ID is available in the context, we can use it, else skip for now and file a Jira ticket for it.